assume.go

```diff
@@ -48,24 +48,24 @@ func assumeRole(account, role string) {
 	role = adjustRoleName(role)
 
 	err := tryToAssumeRole(account, role)
-	if err != nil {
-		needToBeRotatedChan := make(chan bool)
-		go isNeedRotateKey(needToBeRotatedChan)
```

Comment:
The idea here to verify, whether the key must be rotated, during the user is typing his MFA token. This optimization saves a few precious seconds of our lives while we are waiting for the role assumption. If you do it multiple times a day, it's nice to have your role assumed as fast as possible.